### PR TITLE
Tools to create loop mesh

### PR DESCRIPTION
**Pull request by gthyagi**
Hi,

I've added modules in the utilities folder to read Medit files, create DMPlex objects from them, and generate Exodus files. I'll be adding examples to the Underworld documentation repository soon.

Cheers,
Thyagi

### CODE REVIEW

**Comment by julesghub** [comment]
Nice stuff @gthyagi ,
I notice the tests are failing because 
`ModuleNotFoundError: No module named 'read_medit_ascii'`

Do you have a fix for that?

**Comment by gthyagi** [comment]
I fixed the module error.

File: src/underworld3/utilities/read_medit_ascii.py [hunk 1]
```diff
@@ -44,16 +44,16 @@ def open_file(path_or_buf, mode="r"):
 
 def read(filename, mesh_data_name):
```
Comment (julesghub) [inline src/underworld3/utilities/read_medit_ascii.py]: I think you shoudl rename this function too as `read` is quite general - but this function is very specific only for medit files right?

Comment (gthyagi) [reply src/underworld3/utilities/read_medit_ascii.py]: Done! Changed the `read` to `read_medit_ascii`.

**Review by julesghub — APPROVED** [review]

**Pull request merged**